content/integrations/new_relic.md

```diff
@@ -20,22 +20,25 @@ Connect to New Relic to:
 ### New Relic Alerts in Event Stream
 
 1.  On the Webhook tab of New Relic's alerting notification settings page, enter the following webhook URL:
+`https://app.datadoghq.com/intake/webhooknewrelic?api_key={YOUR_DATADOG_API_KEY}`
 
-        https://app.datadoghq.com/intake/webhook/newrelic?api_key={YOUR_DATADOG_API_KEY}
-
-1.  For 'Custom Payload'(s), select JSON 'Payload Type'.
+2.  For 'Custom Payload'(s), select JSON 'Payload Type'.
 
 ### New Relic APM Metric Collection
 
 1.  Locate your API key on New Relic's API Keys page (**Account Settings** -> **Integrations** -> **API Keys**) and enter it in the form on the [Datadog New Relic Integration](https://app.datadoghq.com/account/settings#integrations/new_relic) page.
+  *  *Metrics can only be imported for New Relic customers at the Pro level or above.*
+  * *Make sure you're using the the REST API Key, the User API Keys won't work*
 
-    *Note: Metrics can only be imported for New Relic customers at the Pro level or above.*
-
-1.  If you wish to tag metrics at an account level, please add an account tag.
-1.  Choose whether you want to collect your metrics per hosts or app-wide.
+2.  If you wish to tag metrics at an account level, please add an account tag.
```

Comment:
Also, what does this mean, exactly? I guess it means to tag all metrics with your account number. How about:

"To tag all metrics with your New Relic account number, add an account tag."